DB_PDO.php

```diff
@@ -333,7 +333,10 @@ public function setupConnection()
                 $this->link->setAttribute(PDO::MYSQL_ATTR_USE_BUFFERED_QUERY, true);
             }
         } catch (PDOException $ex) {
-            $this->logger->setErrorMessage('Connection Error: ' . $ex->getMessage());
+            $this->logger->setErrorMessage('Connection Error: ' . $ex->getMessage() .
+                ", DSN=" . $this->dbSettings->getDbSpecDSN() .
+                ", User=" . $this->dbSettings->getDbSpecUser() .
+                ", Password=" . $this->dbSettings->getDbSpecPassword());
```

Comment:
I think password (and username) shouldn't output in error messages for security reasons.